### ISSUE TITLE
Error trying to mix deps.get

### ISSUE THREAD

**Issue by ocodista** [Zatvobor/tirexs#290]
10 1.481 Resolving Hex dependencies...
#10 1.594
#10 1.594 Failed to use "jsx" because
#10 1.594   exjsx (version 3.2.0) requires ~> 2.6.2
#10 1.594   mix.lock specifies 2.11.0
#10 1.594
#10 1.594
#10 1.594 Failed to use "jsx" because
#10 1.594   exjsx (version 3.2.1) requires ~> 2.8.0
#10 1.594   mix.lock specifies 2.11.0
#10 1.594
#10 1.598 ** (Mix) Hex dependency resolution failed, change the version requirements of your dependencies or unlock them (by using mix deps.update or mix deps.unlock). If you are unable to resolve the conflicts you can try overriding with {:dependency, "~> 1.0", override: true}

**Comment by OpakAlex** [comment]
@CAIOHOBORGHI do you need any help?

**Comment by ocodista** [comment]
Yes, I do... I'm trying to use tirexs but when I run mix deps.gets this is the error message thrown /\
